images/python/2.7.Dockerfile

```diff
@@ -25,7 +25,7 @@ ENV TMPDIR=/tmp \
     BASH_ENV=/home/.bashrc
 
 RUN apk add --no-cache --virtual .build-deps \
-      build-base \
+      build-base tar \
```

Comment:
This will need to be added in a separate `apk add` as in here it will be removed once built.